### PR TITLE
Geoserver: mdbw_parking_sites_default: tweak ON_STREET style; delete outdated mdbw_parking_site_default style

### PR DESCRIPTION
**Pull request by richardforsterNVBW**
# Changes
## Point Size
* changed the point size of `ON_STREET` style of `MobiData-BW:mdbw_parking_sites_types` from 6 to 10
## Deletion 
* `MobiData-BW:mdbw_parking_site_default` was renamed `MobiData-BW:mdbw_parking_sites_default`. The outdated sld and xml files were deleted

### CODE REVIEW

File: etc/geoserver/workspaces/MobiData-BW/styles/mdbw_parking_sites_types.sld
Comment (derhuerst) [inline etc/geoserver/workspaces/MobiData-BW/styles/mdbw_parking_sites_types.sld:1]: This style's `Name` is still `mdbw_parking_site_default`. Shouldn't it be `mdbw_parking_sites_default`?

Comment (hbruch) [reply etc/geoserver/workspaces/MobiData-BW/styles/mdbw_parking_sites_types.sld:1]: @richardforsterNVBW as @derhuerst commented already, could you change the style's name also: https://github.com/mobidata-bw/ipl-orchestration/blob/c211d7d00e44e7eddf7dca779bf3df7709bb2bb8/etc/geoserver/workspaces/MobiData-BW/styles/mdbw_parking_sites_types.sld#L8

Comment (richardforsterNVBW) [reply etc/geoserver/workspaces/MobiData-BW/styles/mdbw_parking_sites_types.sld:1]: I am not sure if I rebased or if it is something else. Anyway, I changed the names now as well. In addition, I observed many differences between the sld syntaxes like :se tags in one style and normal tags in the other one. Therefore, I harmonized both styles.

**Review by hbruch — APPROVED** [review]
LGTM

**Pull request merged**